### PR TITLE
[Rule Tuning] Sudoers File Modification

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/872

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/cross-platform/privilege_escalation_sudoers_file_mod.toml [hunk 1]
```diff
@@ -14,14 +14,14 @@ index = ["auditbeat-*", "logs-endpoint.events.*"]
...
 rule_id = "931e25a5-0f5e-4ae0-ba0d-9e94eff7e3a4"
-severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation"]
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Privilege Escalation"]
 type = "query"
 
 query = '''
-event.category:file and event.type:change and file.path:/etc/sudoers
+event.category :"file" and event.type:"change" and file.path:(/etc/sudoers* or /private/etc/sudoers*)
```
Comment (bm11100) [inline rules/cross-platform/privilege_escalation_sudoers_file_mod.toml:24]:
```suggestion
event.category:file and event.type:change and file.path:(/etc/sudoers* or /private/etc/sudoers*)
```

Comment (bm11100) [reply rules/cross-platform/privilege_escalation_sudoers_file_mod.toml:24]: the quotes don't really matter but for consistency with the other KQL rules, we dont use them for those fields.

**Review by bm11100 — APPROVED** [review]
minor tweak, otherwise LGTM

**Review by threat-punter — APPROVED** [review]
👍

**Comment by austinsonger** [comment]
Resolves #712

**Pull request merged**